applications/active_training.py

```diff
@@ -136,7 +140,7 @@ def launch_pbs_jobs(
         #PBS -e {os.path.join(save_path, "out")}
 
         source ~/.bashrc
-        conda activate ptype
+        conda activate guess-casper
```

Comment:
Is there a way to make the environment a configuration option rather than be hard-coded?

Reply:
I can make these all config options?